CMakeLists.txt

```diff
@@ -79,6 +79,30 @@ if(NOT APPLE_IOS AND NOT ANDROID)
 	option(ENABLE_MONOLITHIC_INSTALL "Install everything in single directory on Linux and Mac" OFF)
 endif()
 
+# On Linux, use -DCMAKE_CXX_COMPILER_LAUNCHER=ccache instead.
+# The XCode and MSVC builds each require some more configuration, which is enabled by the following option:
+if(MSVC OR (CMAKE_GENERATOR STREQUAL "Xcode"))
+  option(ENABLE_CCACHE "Speed up recompilation by caching previous compilations" ON)
+endif()
+
+if(ENABLE_CCACHE AND (CMAKE_GENERATOR STREQUAL "Xcode"))
+	find_program(CCACHE ccache REQUIRED)
+	if(CCACHE)
+		# https://stackoverflow.com/a/36515503/2278742
+		# Set up wrapper scripts
+		configure_file(ios/launch-c.in   ios/launch-c)
```

Comment:
Is placing the ccache wrappers in the ios dir alright?

Reply:
I meant that the wrapper is also used when building for macOS, not just iOS.

Reply:
ah yeah, good point. What about `apple` dir?

Reply:
Done